Classes/Feedback/BITFeedbackWindowController.m

```diff
@@ -353,8 +353,10 @@ - (BOOL)canSendMessage {
 }
 
 - (IBAction)sendMessage:(id)sender {
-  [self.manager submitMessageWithText:[self.messageText string] andAttachments:self.attachments];
+  [self.manager submitMessageWithText:[self.messageText string] andAttachments:[[NSArray alloc] initWithArray:self.attachments]];
```

Comment:
If you want, a small syntax improvement could be to use `[NSArray arrayWithArray:self.attachments];` instead